apps/edxapp/templates/cms/_dc_base.yml.j2

```diff
@@ -82,6 +85,44 @@ spec:
         # ImageStream and BuildConfig templates)
         image: "{{ internal_docker_registry }}/{{ project_name }}/edxapp-{{ service_variant }}:{{ edxapp_image_tag }}-live"
         imagePullPolicy: Always
+{% if celery_worker is not defined %}
```

Comment:
Why not doing the opposite ?

`if celery_worker is defined` looks like clearer, no ?

Reply:
Oh yeah :stuck_out_tongue: